schemas/com.mbta.ctd.glides.trips_updated.v1.json

```diff
@@ -181,6 +193,9 @@
           "maxItems": 2,
           "items": { "$ref": "#/$defs/car" }
         },
+        "revenue": {
+          "oneOf": [{ "const": "revenue" }, { "const": "nonrevenue" }]
```

Comment:
Are these references, or should they be `REVENUE` and `NONREVENUE` or neither?

Reply:
We talked synchronously about this. The API uses the string constants "REVENUE" and "NON_REVENUE" ([example](https://api-v3.mbta.com/vehicles?filter%5Brevenue%5D=NON_REVENUE)), which I didn't realize when I arbitrarily chose the lowercase constants. I guess it'd be nice to use a consistent constant, but:
- It's unlikely anyone would be trying to join these data sources together.
- Consumers would probably parse the constant into an internal format anyway.
- Changing it isn't worth making RTR update their PR to use new constants.

So I'll leave the lowercase constants as is.